### ISSUE TITLE
Layers with .PNG & GIF

### ISSUE THREAD

**Issue by pokerpug** [HashLips/hashlips_art_engine#946]
Hi,

I believe this has been asked before in a variety of different way, but just looking to see if anyone has found a solution.

In short I have multiple layers some of which are .GIF files others .PNG. and looking to combine these together. 
E.g. Layer 1 Background - .PNG 
Layer 2 - Face - .GIF
Layer 3 - Body .PNG
Layer 4 - Arms .GIF
*Final output being also a .GIF file.

Any alternative art generate engines or known work around's would be appreciated.

Cheers

**Comment by bolshoytoster** [comment]
@pokerpug no, the library we use for creating gifs - `gif-encoder-2` - does not support loading them.

**Comment by jalagar** [comment]
Hi @pokerpug I was able to build a tool to do exactly this! it takes png layers and turns them into generative gifs. Its forked from an earlier version of hashlips but I plan to update it to a more recent version soon.

Here's a medium article: https://jalagar-eth.medium.com/how-to-create-generative-animated-nft-art-in-under-an-hour-e7dab1785c56
And here's the repo: https://github.com/jalagar/Generative_Gif_Engine
Feel free to follow me on twitter: https://twitter.com/jalagar_eth for more updates.

You can see all 1k of them on https://opensea.io/collection/genesis-bouncing-ball. Let me know if you have any questions!
